### PR TITLE
Update OKD after master moved to 4.8

### PR DESCRIPTION
**Pull request by vrutkovs**
This updates custom configuration for OKD components.

### CODE REVIEW

File: ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  disabled: true
+  name: "4.8"
+  namespace: origin
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
```
Comment (gabemontero) [inline ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml:49]: should this not be 4.8

that would be consistent with https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.7__okd.yaml

and the master file below

**Review by gabemontero — COMMENTED** [review]
on possible need for update

File: ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml (hunk 1 above)
Comment (vrutkovs) [reply ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml]: Fixed, thanks.

File: ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.8-presubmits.yaml [hunk 2]
```diff
@@ -1312,6 +1312,337 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
```
Comment (sinnykumari) [inline ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.8-presubmits.yaml:1315]: 'optional: true` is missing for the jobs

Comment (vrutkovs) [reply ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.8-presubmits.yaml:1315]: Updated:
* https://github.com/openshift/release/pull/15802/files#diff-38238cb6f4ec8238b5e94af12df0ddac3ca1d17cb88e5a1d80c00c0a467c927fR1329
* https://github.com/openshift/release/pull/15802/files#diff-38238cb6f4ec8238b5e94af12df0ddac3ca1d17cb88e5a1d80c00c0a467c927fR1399
* https://github.com/openshift/release/pull/15802/files#diff-38238cb6f4ec8238b5e94af12df0ddac3ca1d17cb88e5a1d80c00c0a467c927fR1471
* https://github.com/openshift/release/pull/15802/files#diff-38238cb6f4ec8238b5e94af12df0ddac3ca1d17cb88e5a1d80c00c0a467c927fR1543
*

**Comment by sinnykumari** [comment]
LGTM

**Comment by smarterclayton** [comment]
/approve

but ooo until tomorrow so can’t review

File: ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    tag: rhel-8-golang-openshift-4.6
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```
Comment (LorbusChris) [inline ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml:30]: shouldn't it be 1.15?

Comment (vrutkovs) [reply ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml:30]: I'm using the same tags as in openshift-cluster-samples-operator-release-4.8.yaml here

File: ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+images:
+- dockerfile_path: Dockerfile.okd
+  from: base
+  inputs:
+    openshift_release_golang-1.13:
+      as:
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
```
Comment (LorbusChris) [inline ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml:37]: @vrutkovs I don't think it uses 1.13 for OCP: https://github.com/openshift/release/blob/8b814ee6fd2fe28631f3acb89c9890ea75fafd0a/ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8.yaml#L36

Comment (vrutkovs) [reply ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml:37]: ah, good catch, updated

**Comment by smarterclayton** [comment]
/hold

if you want to see whether these rehearse

**Comment by vrutkovs** [comment]
4.8 failures are odd (bootstrap broken?), we'll figure those out one by one

/hold cancel

**Pull request merged**